File: beautify_api/8.0.0/aspnet-core/src/BanHangBeautify.Application/NhanSu/NhanVien/Dto/NhanSuDto.cs
using Abp.Application.Services.Dto;
using System;
using System.ComponentModel.DataAnnotations;

namespace BanHangBeautify.NhanSu.NhanVien.Dto
{
    public class NhanSuDto : EntityDto<Guid>
    {
        public string MaNhanVien { get; set; }
        public string Ho { get; set; }
        public string TenLot { get; set; }
        public string TenNhanVien { get; set; }
        public string DiaChi { get; set; }
        public string SoDienThoai { get; set; }
        public string CCCD { get; set; }
        public DateTime NgaySinh { get; set; }
        public int KieuNgaySinh { get; set; }
        public int GioiTinh { get; set; }
        [MaxLength(256)]
        public string NgayCap { get; set; }
        [MaxLength(2000)]
        public string NoiCap { get; set; }
        public byte[] Avatar { get; set; }
        public Guid IdChucVu { set; get; }
    }
}

File: beautify_app/lib/screens/app/nhan_vien/models/NhanSuDto.dart
class NhanSuDto {
  late String? id;
  late String? maNhanVien;
  late String? tenNhanVien;
  late String? diaChi;
  late String? soDienThoai;
  late String? cccd;
  late String ngaySinh;
  late int? kieuNgaySinh;
  late int gioiTinh;
  late String? ngayCap;
  late String? noiCap;
  late String? avatar;
  late String idChucVu;

  NhanSuDto(
      {required this.id,
      required this.maNhanVien,
      required this.tenNhanVien,
      this.diaChi,
      this.soDienThoai,
      this.cccd,
      required this.ngaySinh,
      this.kieuNgaySinh,
      required this.gioiTinh,
      this.ngayCap,
      this.noiCap,
      this.avatar,
      required this.idChucVu});

  NhanSuDto.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    maNhanVien = json['maNhanVien'];
    tenNhanVien = json['tenNhanVien'];
    diaChi = json['diaChi'];
    soDienThoai = json['soDienThoai'];
    cccd = json['cccd'];
    ngaySinh = json['ngaySinh'];
    kieuNgaySinh = json['kieuNgaySinh'];
    gioiTinh = json['gioiTinh'];
    ngayCap = json['ngayCap'];
    noiCap = json['noiCap'];
    avatar = json['avatar'];
    idChucVu = json['idChucVu'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = <String, dynamic>{};
    data['id'] = id;
    data['maNhanVien'] = maNhanVien;
    data['tenNhanVien'] = tenNhanVien;
    data['diaChi'] = diaChi;
    data['soDienThoai'] = soDienThoai;
    data['cccd'] = cccd;
    data['ngaySinh'] = ngaySinh;
    data['kieuNgaySinh'] = kieuNgaySinh;
    data['gioiTinh'] = gioiTinh;
    data['ngayCap'] = ngayCap;
    data['noiCap'] = noiCap;
    data['avatar'] = avatar;
    data['idChucVu'] = idChucVu;
    return data;
  }
}